user-guide/Advanced_Functionality/Databases/Amazon_Keyspaces_Service.md

```diff
@@ -47,6 +47,38 @@ For Amazon Keyspaces, the Starfield certificate must be present on the local Win
 
    ![Add Certificate](~/user-guide/images/aks_add_certificate.png)
 
+## Generating credentials for Amazon Keyspaces
+
+In the Identity and Access Management (IAM) Service, you can generate new credentials that are used to access the Amazon Keyspaces service. Those credentials are then linked to the account that created them.
+
+In order to do this, the right permissions are required.
+
+All of this takes place in the IAM service.
+
+1. Go to *IAM* > *Users*.
```

Comment:
How do you get to IAM? Where should users look to find this?

Reply:
added a direct link